requirements.txt

```diff
@@ -0,0 +1,5 @@
+# Add Biryani extra requirement here because it is needed by OpenFisca-Core and installation from Git URL below
+# does not take extras_require of Core setup.py into account.
+Biryani[datetimeconv] >= 0.10.4
+--editable git+https://github.com/openfisca/openfisca-core.git#egg=OpenFisca-Core
```

Comment:
Why do we need to install `openfisca-core` from git ?

If the package is said to depend on `OpenFisca-Core >= 3.0.0, < 4.0`, I would expect the test to run with a version of core corresponding to this constraint.
 
Otherwise, the test result may be biased. You can have tests passing on Travis, but the application failing to run for real as the version of core would be different.